src/script/src/python/ffi_types/copr.rs

```diff
@@ -343,20 +343,43 @@ pub(crate) enum Either {
     Rb(RecordBatches),
     AffectedRows(usize),
 }
+
+impl PyQueryEngine {
+    pub(crate) fn sql_to_rb(&self, sql: String) -> StdResult<RecordBatch, String> {
+        let res = self.query_with_new_thread(sql.clone())?;
+        match res {
+            Either::Rb(rbs) => {
+                let rb = compute::concat_batches(
+                    rbs.schema().arrow_schema(),
+                    rbs.iter().map(|r| r.df_record_batch()),
+                )
+                .map_err(|e| {
+                    format!(
+                        "Concat batches failed for query {sql}: {e}",
+                        sql = sql,
+                        e = e
+                    )
```

Comment:
```suggestion
                    format!("Concat batches failed for query {sql}: {e}")
```

Reply:
Hi there's copilot's suggestiong! didn't think much and pressed tab XD